gss-cb-signalling.xml

```diff
@@ -70,7 +70,7 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     </section>
     <section title="Channel Binding State Extension" anchor="d1e413">
       <t>
-We propose a new return flag for GSS_Init_sec_context() and GSS_Accept_sec_context(), as well as a pair of functions for a) creating “empty” security context handles, b) requesting flags and indicating which flags the application understands. We also add new mechanism attributes describing mechanism capabilities.</t>
+We propose a new return flag for GSS_Init_sec_context() and GSS_Accept_sec_context(), as well as a pair of functions for a) creating “empty” security context handles, b) requesting flags and indicating which flags the application understands. We also add new mechanism attribute describing mechanism capabilities.</t>
```

Comment:
This sentence needs further changes: "a new mechanism attribute" for starters, and maybe strike "describing mechanism capabilities" since it was vague and doesn't really make sense for a single attribute.